### PR TITLE
allow go update deps to take a domain arg

### PR DESCRIPTION
**Pull request by dprotaso**
Contributes to #2174

### CODE REVIEW

**Comment by dprotaso** [comment]
Does this repo depend on the hack repo now?

**Comment by dprotaso** [comment]
here's the hack PR https://github.com/knative/hack/pull/9

File: scripts/library.sh [hunk 1]
```diff
@@ -517,11 +517,13 @@ function go_update_deps() {
 
   local UPGRADE=0
   local VERSION="master"
+  local DOMAIN="knative.dev"
```
Comment (n3wscott) [inline scripts/library.sh:520]: This file lives in `knative/hack` now

**Comment by dprotaso** [comment]
ugh - i don't know why my diff includes these extra license files - let me remake this

**Comment by dprotaso** [comment]
it's probably my license tool

**Comment by dprotaso** [comment]
fixed my local setup of `go-licenses` closing in favour of https://github.com/knative/test-infra/pull/2551

**Pull request closed without merging**